### PR TITLE
User register and login

### PR DESCRIPTION
**Pull request by Godhanded**
Please let's take a look at the Schema again, there seems to be a problem with the relationships. Please test out this and let's work on it together

### CODE REVIEW

File: Backend/models.py [hunk 1]
```diff
@@ -324,6 +324,7 @@ class UserTransactions(db.Model):
     user = db.relationship('User', backref='users_transactions', lazy=True)
 
 
+
     def __init__(self, id, type, description, amount, status, date, time):
         self.id = id
```
Comment (Tolulophey) [inline Backend/models.py]: remove self.id from all modal classes __init__ functions

Comment (Tolulophey) [reply Backend/models.py]: a lot of modifications had been made to the model.py at my own end though
I now linked the wallet and Details table to User and company and then linked Wallet and Transaction table

File: Backend/flaskr/__init__.py [hunk 2]
```diff
@@ -24,4 +27,62 @@ def after_request(response):
     def hello():
         return "hello world"
 
-    return app
\ No newline at end of file
+    @app.route("/users/register",methods=["POST"])
```
Comment (Tolulophey) [inline Backend/flaskr/__init__.py:33]: change this endpoint to /register, you can use the same endpoint to register both company and user but if you prefer user then make it /users

Comment (Tolulophey) [inline Backend/flaskr/__init__.py:33]: change this endpoint to /register, you can use the same endpoint to register both company and user but if you prefer user then make it /users

**Review by Tolulophey — CHANGES_REQUESTED** [review]

File: Backend/models.py (hunk 1 above)
Comment (Godhanded) [reply Backend/models.py]: Okay let's keep in sync, create a pr with the updated models, we'll merge it then I'll pull to my branch, then update my pr

Comment (Tolulophey) [reply Backend/models.py]: ok

**Review by Tolulophey — APPROVED** [review]
nice work

**Review by Tolulophey — APPROVED** [review]

**Pull request merged**